airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1686,6 +1687,19 @@ definitions:
       type:
         type: string
         enum: [JsonlDecoder]
+  KeysToLower:
```

Comment:
why is `KeysToLower` needed?

Reply:
How would we reference the transformation from the manifest if we don't have that?